kanidm_proto/src/v1.rs

```diff
@@ -51,6 +51,8 @@ pub enum ConsistencyError {
     DuplicateUniqueAttribute(String),
     InvalidSpn(u64),
     SqliteIntegrityFailure,
+    BackendAllIDSSync,
```

Comment:
Clippy's going to hate `BackendAllIDSSync`, can it be BackendAllIdsSync? (or `BackendIdsAllSync` because looking at that in a non-monospace-sans-serif font makes my eyes boggle) :)

Reply:
Fixed :)